docs/docs/Components/components-processing.md

```diff
@@ -193,57 +183,60 @@ This component routes requests to the most appropriate LLM based on OpenRouter m
 | output | Output | The response from the selected model |
 | selected_model | Selected Model | Name of the chosen model |
 
-## Merge Data (Data Combiner)
+## Merge data (Data combiner)
 
-This component combines data using different operations.
+This component combines multiple data sources into a single unified [Data](/concepts-objects#data-object) object.
+
+The component iterates through the input list of data objects, merging them into a single data object. If the input list is empty, it returns an empty data object. If there's only one input data object, it returns that object unchanged. The merging process uses the addition operator to combine data objects.
 
 ### Inputs
 
 | Name | Display Name | Info |
 |------|--------------|------|
-| data_inputs | Data Inputs | Data to combine (minimum 2 inputs required) |
-| operation | Operation Type | Operation to perform (Concatenate/Append/Merge/Join) |
+| data | Data | A list of data objects to be merged |
 
 ### Outputs
 
 | Name | Display Name | Info |
 |------|--------------|------|
-| combined_data | DataFrame | The combined data result |
+| merged_data | Merged Data | A single [Data](/concepts-objects#data-object) object containing the combined information from all input data objects |
+
 
-## Message to Data
+## Message to data
 
-This component converts Message objects to Data objects.
+This component converts [Message](/concepts-objects#message-object) objects to [Data](/concepts-objects#data-object) objects.
 
 ### Inputs
 
 | Name | Display Name | Info |
 |------|--------------|------|
-| message | Message | The Message object to convert to a Data object |
+| message | Message | The [Message](/concepts-objects#message-object) object to convert to a [Data](/concepts-objects#data-object) object |
 
 ### Outputs
 
 | Name | Display Name | Info |
 |------|--------------|------|
-| data | Data | The converted Data object |
+| data | Data | The converted [Data](/concepts-objects#data-object) object |
 
-## Parse Data (Data to Message)
+## Parse Data (Data to message)
 
-This component converts Data objects into Messages using templated formatting.
+The ParseData component converts data objects into plain text using a specified template.
+This component transforms structured data into human-readable text formats, allowing for customizable output through the use of templates.
 
 ### Inputs
 
 | Name | Display Name | Info |
 |------|--------------|------|
-| data | Data | The data to convert to text (can be list) |
-| template | Template | Template for formatting (`{text}`, `{data`, or any key in Data) |
-| sep | Separator | Separator between multiple data items |
+| data | Data | The data to convert to text. |
+| template | Template | The template to use for formatting the data. It can contain the keys `{text}`, `{data}` or any other key in the data. |
```

Comment:
```suggestion
| template | Template | The template to use for formatting the data. It can contain the keys `{text}`, `{data}`, or any other key in the data. |
```